### ISSUE TITLE
Light/Dark Mode Custom Renderer Not Working in IOS 13.4 [Bug] 

### ISSUE THREAD

**Issue by myonlinegolfclub** [xamarin/Xamarin.Forms#10249]
### Description
Since dark mode was supported in IOS it has always worked perfectly using this guide:
https://devblogs.microsoft.com/xamarin/modernizing-ios-apps-dark-mode-xamarin/
In the latest release it no longer works for the top bar. Everything else still uses the theme correctly.  If you start the app in dark mode the top bar loads in light mode. If you minimise the app and open it again it corrects itself.

![image](https://user-images.githubusercontent.com/25149881/78724144-b626c880-7924-11ea-8d19-d87dae0bf0dc.png)

### Steps to Reproduce

1. Setup a blank project
2. Implement the light and dark mode here https://devblogs.microsoft.com/xamarin/modernizing-ios-apps-dark-mode-xamarin/
3. Start the app. The top bar will use Light mode even when the app starts in dark mode.
4. Minimise the app and open it again and it will correct itself

### Expected Behavior
When the app is opened in dark mode the entire app should use the dark theme

### Actual Behavior
The app uses the dark theme but the top bar uses the default light theme

### Basic Information

- Version with issue: 13.4
- Last known good version: Previous

### Screenshots
![image](https://user-images.githubusercontent.com/25149881/78724144-b626c880-7924-11ea-8d19-d87dae0bf0dc.png)

### Workaround
No work around

**Comment by jfversluis** [comment]
Hey there! Thanks for this report!

You're saying

> Last known good version: Previous

That is kind of hard to determine which version that is. Are you using pre-releases? Nightlies? Could you please specify actual version numbers and if possible provide a reproduction project that shows the issue?

**Comment by VladislavAntonyuk** [comment]
I suppose 13.3

**Comment by myonlinegolfclub** [comment]
Hey
I have created a reproduction here: 
https://1drv.ms/u/s!AsbJifSZOv-4lYpsWHabgVBpiUa0aQ?e=oqMcfK

**Reproduction steps**
1. Open the IOS simulator on a mac
2. Set the developer setting 'dark mode' to on
3. Run the project
4. The top bar will load in light theme (incorrect)
5. Minimise and open the app again and it will switch to dark theme
6. It should start in dark theme as the phone is in dark mode

Many thanks
Chris

**Comment by jfversluis** [comment]
@VladislavAntonyuk 
> I suppose 13.3

I was hoping for a Xamarin.Forms version ;)

@myonlinegolfclub thanks for the reproduction! Added it here so we don't lose it.

[LightDark.zip](https://github.com/xamarin/Xamarin.Forms/files/4455946/LightDark.zip)

I don't think is a Forms bug, or a bug at all. Since the detection of the theme is a bit weird you will have to fit it in with the app lifecycle specifically. We want to set the theme as early on as possible. You are using a renderer for `ContentPage` however, the first page that is loaded is a `TabbedPage`. When I make some changes in the renderer and make it into the `TabbedRenderer` things start working. Your updates sample is 
[here](https://github.com/xamarin/Xamarin.Forms/files/4456112/LightDarkWorking.zip).

Basically the only code that I changed are these lines in the `PageRenderer.cs`

```csharp
[assembly: ExportRenderer(typeof(TabbedPage), typeof(LightDark.iOS.PageRenderer))]
namespace LightDark.iOS
{
    public class PageRenderer : Xamarin.Forms.Platform.iOS.TabbedRenderer
```

I'm closing this issue for now, if you think this is in error please let us know.

Also, there is better support coming for OS themes and theme detection in Forms 4.6 and today you can use Essentials for that :)

Thanks!
